src/test/java/HellosTest.java

```diff
@@ -19,7 +18,7 @@ public void setUp() {
   @Test
   public void testLineStructure() {
 
-    assertEquals("Welcome line isn't correct.", Hellos.WELCOME_LINE, lines[0]);
+    assertEquals(Hellos.WELCOME_LINE, lines[0], "Welcome line isn't correct.");
```

Comment:
Fixing this argument ordering problem alone is worth moving to Junit 5 – thanks. :)